### PR TITLE
YEP-170-Adding-the-pictures-system

### PR DESCRIPTION
**Pull request by alexis-moins**
- feat(back): added route /static to server static files
- feat(prisma): added migration for profile_picture fields in tables artist and host
- feat: added route /profiles/profile-picture
- chore(back): updated README

### CODE REVIEW

File: back/prisma/migrations/20230704090316_add_profile_picture_in_artist_and_host/migration.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+-- AlterTable
+ALTER TABLE `artist` ADD COLUMN `profile_picture` LONGTEXT NULL;
```
Comment (Xerataw) [inline back/prisma/migrations/20230704090316_add_profile_picture_in_artist_and_host/migration.sql:2]: Longtext?

File: back/prisma/migrations/20230704090316_add_profile_picture_in_artist_and_host/migration.sql [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
+-- AlterTable
+ALTER TABLE `host` ADD COLUMN `profile_picture` TEXT NULL;
```
Comment (Xerataw) [inline back/prisma/migrations/20230704090316_add_profile_picture_in_artist_and_host/migration.sql:5]: Text ? Or longText, i'm confused....

Comment (alexis-moins) [reply back/prisma/migrations/20230704090316_add_profile_picture_in_artist_and_host/migration.sql:5]: Done

**Review by Xerataw — APPROVED** [review]
Perfect

**Comment by filveith** [comment]
@alexis-moins Looser 👎

**Pull request merged**